File: JavaScript/1012.js
var input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8');
var lines = input.split('\n');

const pegarValores = (line) => line.split(" ").map(a=>+a)
const[a,b,c] = pegarValores(lines.shift())

let pi = 3.14159
let areaT = (a*c)/2;
let areaC = pi*(c*c)
let areaTr = ((a+b)*c)/2
let areaQr = b*b
let areaR = a*b
console.log(`TRIANGULO: ${areaT.toFixed(3)}`) 
console.log(`CIRCULO: ${areaC.toFixed(3)}`) 
console.log(`TRAPEZIO: ${areaTr.toFixed(3)}`) 
console.log(`QUADRADO: ${areaQr.toFixed(3)}`) 
console.log(`RETANGULO: ${areaR.toFixed(3)}`)

File: Python/1012.py
lista = (input().split(" "))
a,b,c = lista
tri = (float(a)*float(c))/2
cir = (float(c)*float(c))*3.14159
tra = ((float(a)+float(b))*float(c))/2
qua = (float(b)*float(b))
ret = (float(a)*float(b))
print("TRIANGULO: %0.3f"%tri)
print("CIRCULO: %0.3f"%cir)
print("TRAPEZIO: %0.3f"%tra)
print("QUADRADO: %0.3f"%qua)
print("RETANGULO: %0.3f"%ret)